src/quotamodel.c

```diff
@@ -1249,13 +1252,10 @@ flush_to_table_size(void)
 		}
 	}
 
-	if (delete_entries_num) delete_from_table_size_map(delete_statement.data);
-	if (insert_entries_num) insert_into_table_size_map(insert_statement.data);
+	if (delete.tableids) delete_from_table_size_map(delete.tableids, delete.segids);
+	if (update.tableids) update_table_size_map(update.tableids, update.sizes, update.segids);
 
```

Comment:
Where is the memory freed after using tableids, sizes and segids?

Reply:
[here](https://github.com/arenadata/gpdb/blob/3069a806b39b5999fea972764c3165e0b075e745/src/backend/utils/adt/arrayfuncs.c#L4819-L4821)